### ISSUE TITLE
Sitecore 9.2 support

### ISSUE THREAD

**Issue by markgibbons25** [bartverdonck/Sitecore-Forms-Extensions#49]
Greetings, and thank you to everyone that has contributed to this great module.
I'd like to ask if anyone has considered what needs to change for proper Sitecore 9.2 support?

The main issue I can see is the built-in functionality of form field to EXM token mapping. Can the current release for 9.1 work alongside or does it conflict? I'll test this myself some time in the next few days.

If anyone is interested in coordinating development with 9.2 support please +1. Thanks!

**Comment by bartverdonck** [comment]
Hi Mark,

I have tested the module on 9.2 and there were no conflicts.

It's indeed the case that the default send mail action of Sitecore has been updated to support tokens. However, the default Sitecore sendmail action has not yet support to mail to a backoffice. It can also be very cumbersome to map each field through the editor and then again in your exm message. (There is no catch-all, or automapping) Finally for backwards compatibility reason, I plan to keep the send mail action of the form extensions as-is.

Best Regards,

Bart
